pennylane/pulse/transmon.py

```diff
@@ -257,23 +257,30 @@ def transmon_drive(amplitude, phase, freq, wires, d=2):
 
     .. math::
 
-        \Omega(t) \left(e^{i (\phi(t) + \nu t)} b_q + e^{-i (\phi(t) + \nu t)} b^\dagger_q\right)
+        \Omega(t) \sin\left(\phi(t) + \nu t\right) \sum_q Y_q
 
-    where :math:`[b^\dagger_p, b_q] = i \delta_{pq}` are creation and annihilation operators
-    and :math:`q` is the qubit label (``wires``).
+    where :math:`Y_q` are Pauli-Y operators on ``wires`` :math:`\{q\}`.
     The arguments ``amplitude``, ``phase`` and ``freq`` correspond to :math:`\Omega / (2\pi)`, :math:`\phi`
     and :math:`\nu / (2\pi)`, respectively, and can all be either fixed numbers (``float``) or depend on time
     (``callable``). If they are time-dependent, they need to abide by the restrictions imposed
     in :class:`ParametrizedHamiltonian` and have a signature of two parameters, ``(params, t)``.
 
+    Together with the qubit :math:`Z` terms in :func:`transmon_interaction`, driving with this term can generate
+    :math:`X` and :math:`Y` rotations by setting :math:`\phi` accordingly and driving on resonance
+    (see eqs. (79) - (92) in `1904.06560 <https://arxiv.org/abs/1904.06560>`_).
+    Further, it can generate entangling gates by driving at cross-resonance with a coupled qubit
+    (see eqs. (131) - (137) in `1904.06560 <https://arxiv.org/abs/1904.06560>`_). Such a coupling is described in :func:`transmon_interaction`.
```

Comment:
```suggestion
    (see eqs. (131) - (137) in `1904.06560 <https://arxiv.org/abs/1904.06560>`_).
    Such a coupling is described in :func:`transmon_interaction`.
```

Reply:
Do you want an explicit line break i.e. also in the rendered docs or just for better readability in the code?

Reply:
Just for code :)